examples/aws-nodejs/index.js

```diff
@@ -108,7 +109,11 @@ app.get('/sts', (req, res, next) => {
     })
   }, next)
 })
-app.post('/sign-s3', (req, res, next) => {
+app.get('/s3/params', (req, res, next) => {
```

Comment:
I wonder if it would be worth renaming the Companion endpoint `/s3/params` is much less clear than `/sign-s3` IMO

Reply:
Unfortunately Companion always has to be backwards compatible as old Uppy clients might connect to Transloadit. So I'm a little more conservative with renames and such.

Reply:
I guess we could support both

Reply:
I don't think we can ever break this api (or else we cannot upgrade transloadit's companion server without breaking clients). i don't want to indefinitely have two endpoints doing the same thing with different names.